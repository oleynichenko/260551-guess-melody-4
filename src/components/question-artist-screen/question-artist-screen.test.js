import QuestionArtistScreen from "@components/question-artist-screen/question-artist-screen";

const question = {
  type: `artist`,
  song: {
    artist: `Jim Beam`,
    src: `path`,
  },
  answers: [
    {
      picture: `path`,
      artist: `John Snow`,
    },
    {
      picture: `path`,
      artist: `Jack Daniels`,
    },
    {
      picture: `path`,
      artist: `Jim Beam`,
    }
  ],
};

describe(`QuestionArtistScreen`, function () {
  it(`should render QuestionArtistScreen with question`, function () {
    const tree = renderer.create(
        <QuestionArtistScreen
          question={question}
          onAnswer={() => {}}
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
