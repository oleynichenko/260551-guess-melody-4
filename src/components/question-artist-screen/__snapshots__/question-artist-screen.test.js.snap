// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QuestionArtistScreen should render QuestionArtistScreen with question 1`] = `
<section
  className="game__screen"
>
  <h2
    className="game__title"
  >
    Кто исполняет эту песню?
  </h2>
  <div
    className="game__track"
  >
    <div
      className="track"
    />
  </div>
  <form
    className="game__artist"
  >
    <div
      className="artist"
    >
      <input
        className="artist__input visually-hidden"
        id="answer-0"
        name="answer"
        onChange={[Function]}
        type="radio"
        value="answer-0"
      />
      <label
        className="artist__name"
        htmlFor="answer-1"
      >
        <img
          alt="John Snow"
          className="artist__picture"
          src="path"
        />
        John Snow
      </label>
    </div>
    <div
      className="artist"
    >
      <input
        className="artist__input visually-hidden"
        id="answer-1"
        name="answer"
        onChange={[Function]}
        type="radio"
        value="answer-1"
      />
      <label
        className="artist__name"
        htmlFor="answer-1"
      >
        <img
          alt="Jack Daniels"
          className="artist__picture"
          src="path"
        />
        Jack Daniels
      </label>
    </div>
    <div
      className="artist"
    >
      <input
        className="artist__input visually-hidden"
        id="answer-2"
        name="answer"
        onChange={[Function]}
        type="radio"
        value="answer-2"
      />
      <label
        className="artist__name"
        htmlFor="answer-1"
      >
        <img
          alt="Jim Beam"
          className="artist__picture"
          src="path"
        />
        Jim Beam
      </label>
    </div>
  </form>
</section>
`;
